seatunnel-e2e/seatunnel-transforms-v2-e2e/seatunnel-transforms-v2-e2e-part-2/src/test/resources/sql_transform/sql_complex_type.conf

```diff
@@ -0,0 +1,100 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+######
+###### This config file is a demonstration of streaming processing in seatunnel config
+######
+
+env {
+  execution.parallelism = 1
+  job.mode = "BATCH"
+}
+
+source {
+    FakeSource {
+      result_table_name="t1"
+      schema = {
+        fields {
+          c_string = "string"
+          c_array = "array<string>"
+          c_map = "map<string,string>"
+          c_row {
+            c_int = "int"
+            c_string = "string"
+            next_row {
+              c_int = "int"
+              c_string = "string"
+            }
+          }
+        }
+      }
+      rows = [
+        {
+          kind = INSERT
+          fields = ["cs",["ca1","ca2","ca3"],{"name":"zs","age":"18"},[1,"sh",[2,"bj"]]]
+        }
+      ]
+    }
+}
+
+transform {
+  Sql {
+    source_table_name = "t1"
+    result_table_name = "t2"
+    query = "select c_string as c1,c_array[0] as c2,c_map['name'] as c3,c_row.next_row.c_string as c4 from t1"
```

Comment:
Can you add this example to document `docs/en/transform-v2/sql.md`?

Reply:
ok, I'll try it